### ISSUE TITLE
Some questions about training. How to save best model? How to load pretrain model?

### ISSUE THREAD

**Issue by f771216203** [WXinlong/SOLO#142]
How to save the best model by min loss?
How to load pretrain model to continue training?

Thanks for answer!

**Comment by WXinlong** [comment]
@f771216203  Usually we directly use the last model, which is the best model in most cases. For loading pre-trained model, you can specify the `load_from` or  `resume_from`  in config file.  The difference is that the latter continues training which means it loads the training statistics.

**Comment by f771216203** [comment]
Thanks for your answer!
